server/models/dummyData/generateListingsData.js

```diff
@@ -1,36 +1,91 @@
 ///////////////////// LISTINGS DATA ////////////////////////////
 //// Generate 100 points within a polygon of Central London ////
 
-const randomPointsOnPolygon = require('random-points-on-polygon');
-const turf = require('turf');
-const faker = require('faker');
+const randomPointsOnPolygon = require("random-points-on-polygon");
+const turf = require("turf");
+const faker = require("faker");
+const neighbs = require('./neighbsData.js').neighbsArray;
+const generatedLandmarks = require('./generateLandmarksData.js');
+const landmarks = generatedLandmarks.landmarksData;
 
 
-var polygon = turf.polygon([[
-  [-.2451, 51.5194], [-.1960, 51.5485],
-  [-.1040, 51.5485], [-.0700, 51.5331],
-  [-.0632, 51.5132], [-.0748, 51.5089],
-  [-.1095, 51.5128], [-.1218, 51.5108],
-  [-.1281, 51.4972], [-.1428, 51.4867],
-  [-.1665, 51.4869], [-.1850, 51.4811],
-  [-.2451, 51.5194]
-]]);
+var polygon = turf.polygon([
+  [
+    [-0.2451, 51.5194],
+    [-0.196, 51.5485],
+    [-0.104, 51.5485],
+    [-0.07, 51.5331],
+    [-0.0632, 51.5132],
+    [-0.0748, 51.5089],
+    [-0.1095, 51.5128],
+    [-0.1218, 51.5108],
+    [-0.1281, 51.4972],
+    [-0.1428, 51.4867],
+    [-0.1665, 51.4869],
+    [-0.185, 51.4811],
+    [-0.2451, 51.5194]
+  ]
+]);
 
 let numberOfPoints = 100;
 var points = randomPointsOnPolygon(numberOfPoints, polygon);
 
 let listingsCoords = [];
 
-
 for (let i = 0; i < points.length; i++) {
   // reverse order to lat-long instead long-lat
-  let latLong = [points[i].geometry.coordinates[1], points[i].geometry.coordinates[0]]
-  listingsCoords.push(latLong)
+  let latLong = [
+    points[i].geometry.coordinates[1],
+    points[i].geometry.coordinates[0]
+  ];
+  listingsCoords.push(latLong);
 }
 
+const client = require("../../../database-redis/index.js");
+
+
+// Redis data load
+(async () => {
+	for (var i = 0; i < 10000; i++) {
```

Comment:
The indentation in this part of the code is showing up as very large tabs on Github -- not sure if it looks different on your computer. The rest of the code is tabbing with two spaces.

Reply:
I'll check it out